optimizer/lib/transformers/ReorderHeadTransformer.js

```diff
@@ -157,6 +159,15 @@ class HeadNodes {
       this._linkIcons.push(node);
       return;
     }
+
+    if (rel === 'preload' ||
+      rel === 'prefetch' ||
+      rel === 'dns-prefetch' ||
+      rel === 'preconnect') {
```

Comment:
nit: I'd prefer the different values in a constant HashMap. Feel free to ignore though...

Reply:
This is in my upper bound of tolerance too. But I opted to keep the same style as the block above. If we add more, then we definitely need to change to improve readability.